### ISSUE TITLE
 The document about the code of FLIR training

### ISSUE THREAD

**Issue by kyloyang1** [Jamie725/Multimodal-Object-Detection-via-Probabilistic-Ensembling#17]
May I ask if the document about the code of FLIR training can be written in more detail? Can these demos be trained with other data sets?

**Comment by Jamie725** [comment]
Yes, you can register your own dataset by modifying codes here: https://github.com/Jamie725/Multimodal-Object-Detection-via-Probabilistic-Ensembling/blob/312da86ac7e6354a2c69c1bfdc5e15a833b9620a/demo/FLIR/demo_train_FLIR.py#L32
